src/features/user/TreeMapper/components/Map.tsx

```diff
@@ -252,7 +234,7 @@ export default function MyTreesMap({
     >
       {satellite && plIds && <SatelliteLayer beforeId={plIds[0]} />}
       {locations &&
-        locations.map((pl: PlantLocation) => {
+        locations.map((pl: Intervention) => {
           const newPl = pl.geometry;
```

Comment:
1. The use of `pl` lacks context after refactoring the code to replace plantLocation with intervention
2. Resolve the TS warning here

Argument of type '(pl: Intervention) => React.JSX.Element | undefined' is not assignable to parameter of type '((value: SampleIntervention, index: number, array: SampleIntervention[]) => Element | undefined) & ((value: Intervention, index: number, array: Intervention[]) => Element | undefined)'.